src/css/main.less

```
@@ -1165,9 +1165,10 @@ dialog {
     min-height: 65px;
 }
 .img_sponsor {
-    height: 100%;
-    max-height: 50px;
+    object-fit: contain;
     margin: 5px 15px;
+    width: 150px;
+    height: 50px;
```

```suggestion
    width: 150px;
    height: 45px;
```
The aspect ratio of the logos is 800/160 (=150/45)

---

Thanks. Fixed.

---

I think now we also need to adjust line 1165 to
```js
min-height: 60px;
```

And limit the max-height in `.img_sponsor` to 45px

---

Done.